### PR TITLE
added homework files for week1-js3

### PR DESCRIPTION
**Pull request by TITHI1244**

### CODE REVIEW

File: javascript/javascript3/week1/cool_api.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+function onChange(event) {
+    userInput = event.target.value;
+}
+input.addEventListener("input", onChange);
+
+function showCurrentWeather() {
+    if(userInput === "") {
+        alert("Please type your city name first...");
+    } else {
+        document.getElementById("input-div").style.display = "none";
```
Comment (aleksd-dfds) [inline javascript/javascript3/week1/cool_api.js:13]: Very long code with repetitive bits. It's hard to read and understand. Your functions should generally not be more than 1 page long, and if they are, it's a sign that you should split it up into more functions.

File: javascript/javascript3/week1/cool_api.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            mainDiv.appendChild(hrElement2);
+
+            const infoDiv = document.createElement("div");
+            mainDiv.appendChild(infoDiv);
+
+            const currentTemp = document.createElement("h3");
+            currentTemp.innerText = `Current temperature: ${temp.innerText}`;
+            infoDiv.appendChild(currentTemp);
+            const feelsLike = document.createElement("h3");
+            feelsLike.innerText = `Feels like: ${Math.floor(response.main.feels_like - 273.15)}°`;
```
Comment (aleksd-dfds) [inline javascript/javascript3/week1/cool_api.js:47]: for example the expression `${Math.floor(x- 273.15)}` is used in many places, which means is is probably a good idea to make a function out of it (with `x` being a parameter)

File: javascript/javascript3/week1/cool_api.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            const maxTemp = document.createElement("h3");
+            maxTemp.innerText = `Maximum today: ${Math.floor(response.main.temp_max - 273.15)}°`;
+            infoDiv.appendChild(maxTemp);
+            const minTemp = document.createElement("h3");
+            minTemp.innerText = `Minimum today: ${Math.floor(response.main.temp_min - 273.15)}°`;
+            infoDiv.appendChild(minTemp);
+
+            const sunrise = document.createElement("h3");
+            const sunriseTime = new Date(response.sys.sunrise * 1000);
+            sunrise.innerText = `Sunrise: ${sunriseTime.getHours()}:${sunriseTime.getMinutes()}:${sunriseTime.getSeconds()}`;
```
Comment (aleksd-dfds) [inline javascript/javascript3/week1/cool_api.js:59]: same for making a string out of hours:minutes:seconds

**Review by aleksd-dfds — APPROVED** [review]
Seems to be working but make sure you go thru the exercise of making the code more readable according to my comments.

File: javascript/javascript3/week1/cool_api.js (hunk 3 above)
Comment (aleksd-dfds) [reply javascript/javascript3/week1/cool_api.js:59]: Never copy / paste if you can avoid it. Reuse by using functions instead.

File: javascript/javascript3/week1/cool_api.js (hunk 1 above)
Comment (TITHI1244) [reply javascript/javascript3/week1/cool_api.js:13]: Yes, got that. Will try to break that into smaller chunks.

File: javascript/javascript3/week1/cool_api.js (hunk 2 above)
Comment (TITHI1244) [reply javascript/javascript3/week1/cool_api.js:47]: Yes. Will do that.

File: javascript/javascript3/week1/cool_api.js (hunk 3 above)
Comment (TITHI1244) [reply javascript/javascript3/week1/cool_api.js:59]: Ok.

**Pull request merged**